### ISSUE TITLE
Master: Chimney need connection to pipes.

### ISSUE THREAD

**Issue by elfl0rd** [StrangeLoopGames/EcoIssues#13561]
To get the smoke out of the inside of the building
e.g. something like this
![изображение](https://user-images.githubusercontent.com/4980243/68122371-3d59b580-ff1b-11e9-8b27-7bba8f83153f.png)

(currently - not working)
![изображение](https://user-images.githubusercontent.com/4980243/68122404-51051c00-ff1b-11e9-8f8a-c9d6e0e45429.png)

**Comment by elfl0rd** [comment]
or add something like "wall piping" block allow pipes pass trough walls horizontally or vertically.

**Comment by keegan-o** [comment]
I can make art for the pipe builders to be able to detect chimneys and funnel into them in a way that looks good-- but connecting pipes of different types is currently not allowed, and should not be allowed for pipes of the same type. 

Connecting chimneys and pipes seems like something you should be able to do, but it will need support in code-- and then-- if you can do that, there will need to be something preventing then connecting with another type of pipe with something else in it, I imagine. 

Or perhaps you can and should be allowed to funnel multiple different pipes into one chimney at different elevations as long as all the pipes are sending CO2?

This will need some design.

The remaining problem is of course the no horizontal chimneys thing-- which I think allowing might end up being the easiest way to effectively get people their 'through the wall piping'. That would require some new art as well-- any solution to that problem would, though.

**Comment by johnkslg** [comment]
Propose pushing this @beastyn

**Comment by beastyn** [comment]
Its kind a pity that we have such an atmospheric block but can't make it useful. But yeah, nothing crtitcal, so pushing to 9.1 for now (we can just remove it for now, so it won't confuse people).

**Comment by elfl0rd** [comment]
No, don't need to remove it :) it's very good in combination with a fireplace.

**Comment by WeaselDog** [comment]
Chimney are now mandatory for progression. We need them for Cast Iron Stove & Bakery Oven which must be inside. It would be really nice to have a block in-place of some the pipes.

Current setup:
![ChimneyInternal](https://user-images.githubusercontent.com/47330823/72224843-a1a87f00-3533-11ea-9ebd-16b1c76c03e7.png)
![ChimneyExternal1](https://user-images.githubusercontent.com/47330823/72224928-75d9c900-3534-11ea-9c50-05d59b9785dc.png)
![ChimneyExternal2](https://user-images.githubusercontent.com/47330823/72224929-78d4b980-3534-11ea-9052-d60dc3f0418f.png)

**Comment by WeaselDog** [comment]
This picture explains what I mean better. If a chimney allows gas to pass through as if it was empty space, it can be used as exhaust without being a pipe extension. The gas will not enter a second pipe at the top of the chimney as shown in this example with empty space:

![Openpipe](https://user-images.githubusercontent.com/47330823/72225361-948e8e80-3539-11ea-97d2-ffac3b3e371a.png)
![Exaust](https://user-images.githubusercontent.com/47330823/72225357-922c3480-3539-11ea-8ce5-33d75300bd55.png)

**Comment by WeaselDog** [comment]
Related issue: https://github.com/StrangeLoopGames/EcoIssues/issues/15302

**Comment by IcedForge** [comment]
Also want to add into this that we currently have 0 Tier 4 pipe capacity, so there is no way to make a "Tier 4" house it always degrades to 3.7-3.9# because i want to use non merging pipes forces me to use T2 + T3 pipes. A possible upgraded copper lined steel pipe?

**Comment by D3nnis3n** [comment]
Bump?
